pkgs/love2d/example/assets/pocket_bomber/src/player.lua

```
@@ -0,0 +1,277 @@
+
+    for _, corner in ipairs(corners) do
+        local col, row = Grid.worldToTile(grid, corner[1], corner[2])
+        if Grid.isSolid(grid, col, row) then
+            return true
+        end
+
+        -- Check bomb collision
+        for _, bomb in ipairs(G.bombs) do
+            if bomb.col == col and bomb.row == row then
+                -- Allow passing through the bomb we're standing on
+                if self.standingOnBomb == bomb then
```

Luacheck (1.2.0)

[warning] empty if branch

(W542)